lib/rack/test/uploaded_file.rb

```diff
@@ -18,8 +19,16 @@ class UploadedFile
       # The content type of the "uploaded" file
       attr_accessor :content_type
 
+      # Creates a new UploadedFile instance.
+      #
+      # @param content [IO, Pathname, String, StringIO] a path to a file, or an {IO} or {StringIO} object representing the
```

Comment:
`@param content [IO, Pathname, String, StringIO] a path to a file`
Is this correct?
For example, IO is used for both a path to a file and file object, right?

Reply:
`Pathname` does not derive from `IO`, or how do you mean?